larq/quantizers.py

```diff
@@ -718,9 +732,11 @@ def get(identifier):
     if identifier is None:
         return None
     if isinstance(identifier, dict):
-        return deserialize(identifier)
+        use_legacy_format = "module" not in identifier
+        return deserialize(identifier, use_legacy_format=use_legacy_format)
     if isinstance(identifier, str):
-        return deserialize(str(identifier))
+        config = {"class_name": str(identifier), "config": {}}
+        return get(config)
```

Comment:
This applies the changes following [this commit](https://github.com/keras-team/keras/commit/7eb8ef28155f20cbd78e1e24b6da5cfb656d4164#diff-e69484e2f50471a5661ca897bd7ae336ea254a351904437d6ecdc6ab67ccd659R201-R207).